packages/edit-site/src/components/sidebar/header/index.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * WordPress dependencies
+ */
+import { Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { useDispatch } from '@wordpress/data';
+import { store as interfaceStore } from '@wordpress/interface';
+
+/**
+ * Internal dependencies
+ */
+import { STORE_NAME } from '../../../store/constants';
+import { SIDEBAR_BLOCK, SIDEBAR_TEMPLATE } from '../constants';
+
+const SettingsHeader = ( { sidebarName } ) => {
```

Comment:
Nitpick: what about calling this `SidebarHeader`?
I was thrown off by the difference between component name and file name (`sidebar/header`).

In the same vein, it would also be fine to rename the folder to `sidebar/settings-header` (but imho it's less clear).

Reply:
I renamed the folder to `settings-heade`. `edit-post` is using `sidebar/settings-header` as well. `SettingsSidebar` is only used together with the default (settings) sidebar (it's not used with the global styles sidebar). That's why I'd keep the `Settings` in the name.

Reply:
I think it makes sense to align the naming with `edit-post` here.